xml/operating_system_darwin.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Patterns for common names of Mac OS X operating systems.
+-->
+<fingerprints matches="operating_system.name">
+  <!-- Match Mac OS Classic first due to weak matching on Mac OS X -->
+  <fingerprint pattern="^(?i:(?:Apple )?Mac OS ([7-9](?:\.\d+?)*?))$">
```

Comment:
This is classic (pre-OS X) Mac OS. I've never heard anything referred to as Apple OS (there was an Apple DOS and of course iOS, but that's it). If anything, we should match on "System 7" -- it wasn't actually called Mac OS until version 7.6. Oh the good old days.